src/platforms/macos/daemon/macosdaemon.cpp

```diff
@@ -14,9 +14,14 @@
 #include <QJsonValue>
 #include <QLocalSocket>
 #include <QProcess>
+#include <QSettings>
 #include <QTextStream>
 #include <QtGlobal>
 
+constexpr const char* LATEST_IPV4GATEWAY = "latestServerIpv4Gateway";
```

Comment:
why isn't this a QString?

Reply:
No big reasons. These strings are used in `QSettings::setValue` (https://doc.qt.io/qt-5/qsettings.html#setValue) as first argument and QString() will be created from `const char*`.